boards/posix/native_posix/native_rtc.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2018 Oticon A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * API to the native_posix (Real) Time Clock
+ */
+
+
+#ifndef _NATIVE_POSIX_RTC_H
+#define _NATIVE_POSIX_RTC_H
+
+#include "hw_models_top.h"
+#include <stdbool.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "zephyr/types.h"
+
+#define RTC_CLOCK_BOOT               0 /* Time since boot, cannot be offset */
+#define RTC_CLOCK_REALTIME           1 /* Persistent clock */
+/* Pseudo-host real time clock (Please see documentation): */
+#define RTC_CLOCK_PSEUDOHOSTREALTIME 2
+
+/**
```

Comment:
Doxygen documentation missing from functions in this file

Reply:
I don't think this files will be even passed thru doxygen, or?. Nevertheless I added the tags in case it would ever be.

Reply:
> I don't think this files will be even passed thru doxygen, or?. Nevertheless I added the tags in case it would ever be.

I have noticed that even if the code is not meant to be run through doxygen, it is useful to add proper documentation to function prototypes in order to clarify the function parameters and usage for anyone reading the code.